README.md

```diff
@@ -50,6 +51,8 @@ And command to create configmap from it:
 kubectl create configmap aet-dictionary --from-file=AETs.json
 ```
 
+The AET dictionary JSON can also be specified directly via the "--aet_dictionary_inline" parameter.
+
 Note that any C-FIND query on the ModalitiesInStudy tag will result in 1 QIDO-RS query per modality.
```

Comment:
Can you move this note to above the "Available AET destinations" section? Now that this section is a bit long, it makes more sense for it to be near where we describe the supported services.

Reply:
Ok